Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/Cards/TwitterCardPlayer.cs

```diff
@@ -92,15 +88,41 @@ protected override void Validate()
         {
             base.Validate();
 
+            if (string.IsNullOrEmpty(this.TwitterSiteUsername))
+            {
+                throw new ArgumentNullException(nameof(this.TwitterSiteUsername));
+            }
+
             if (this.Image == null)
             {
                 throw new ArgumentNullException(nameof(this.Image));
             }
 
+            if (string.IsNullOrEmpty(this.Title))
```

Comment:
This property is required in this case, I also reflected this in the [chart](https://github.com/ASP-NET-MVC-Boilerplate/Framework/issues/3).  You can confirm this requirement [here](https://dev.twitter.com/cards/types/player).